src/measure_theory/covering/differentiation.lean

```diff
@@ -26,6 +28,13 @@ ratio really makes sense.
 For concrete applications, one needs concrete instances of Vitali families, as provided for instance
 by `besicovitch.vitali_family` (for balls) or by `vitali.vitali_family` (for doubling measures).
 
+Specific applications to Lebesgue density points and the Lebesgue differentiation theorem are also
+derived:
+* `vitali_family.ae_tendsto_measure_inter_div` states that, for almost every point `x ∈ s`,
+  then `μ (s ∩ a) / μ a` tends to `1` as `a` shrinks to `x` along a Vitali family.
```

Comment:
```suggestion
  `μ (s ∩ a) / μ a` tends to `1` as `a` shrinks to `x` along a Vitali family.
```

Reply:
That's a (maybe a little bit old-fashioned) rule in mathematics writing that two sequences of symbols should always be separated by at least one word, to make sure that it is not mistaken for one bigger formula. That's why I put a "then" here.